app-text/sigil/sigil-0.9.9.recipe

```diff
@@ -0,0 +1,98 @@
+SUMMARY="A free, open source, multi-platform ebook editor"
+DESCRIPTION="Sigil is designed to edit books in ePub format (both ePub 2 and \
+ePub 3)."
+HOMEPAGE="https://sigil-ebook.com/"
+COPYRIGHT="2009, 2010, 2011 Strahinja Markovic
+	2011, 2012 John Schember
+	2012 Dave Heiland
+	2012 John Schember"
```

Comment:
Oops, I just undestood what happened. The [diff](https://github.com/haikuports/haikuports/pull/2117#pullrequestreview-116764124) @alaviss wrote did not cover the full `COPYRIGHT`, and you ate the [next two lines](https://github.com/haikuports/haikuports/pull/2117/commits/dbb7404828f7a4e5bf0d7eae4d3dcf3490a66efb#diff-ed6ed231d7f69ac1e469f24e27f14a71R9) when you made the changes.
```
	2012 Grant Drake
	2016 Kevin B. Hendricks"
```
Then I noticed the missing quotes and thought you had eaten the closing quotes after `2012 John Schember`.